src/Input/Input.js

```diff
@@ -130,7 +130,7 @@ export default class Input extends Component {
     /**
      * The input value, required for a controlled component.
      */
-    value: PropTypes.string,
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
It's only used in one other place, so easily done. I guess it will go away with `next`...